core/domain/user_services_test.py

```diff
@@ -86,6 +86,26 @@ def test_set_and_get_username(self):
         self.assertEqual(
             username, user_services.get_username(user_settings.user_id))
 
+    def test_set_username_existing_username_raises_error(self):
```

Comment:
test_set_username_to_existing_username_raises_error

(try to make the sentence "scan")